test/service/test_manifest.py

```diff
@@ -1195,6 +1199,43 @@ def test_full_metadata_missing_fields(self):
         self.assertGreater(len(intersection), 0)
         self.assertGreater(len(symmetric_diff), 0)
 
+    @manifest_test
+    def test_curl_format_manifest(self):
+        self.maxDiff = None
+        bundle_fqid = BundleFQID('f79257a7-dfc6-46d6-ae00-ba4b25313c10', '2018-09-14T133314.453337Z')
+        self._index_canned_bundle(bundle_fqid)
+        filters = {'fileFormat': {'is': ['pdf']}}
+        response = self._get_manifest(ManifestFormat.curl, filters)
+        self.assertEqual(200, response.status_code)
+        lines = response.content.decode('utf-8').splitlines()
+        header, body = lines[:4], lines[4:]
+        expected_header = [
+            '--create-dirs',
+            '',
+            '--compressed',
+            '',
+        ]
+        self.assertEqual(expected_header, header)
+        expected_body = [
+            [
+                f'url="{config.service_endpoint()}/repository/files/0db87826-ea2d-422b-ba71-b15d0e4293ae"',
+                'output="SmartSeq2_sequencing_protocol.pdf"',
+                ''
+            ],
+            [
+                f'url="{config.service_endpoint()}/repository/files/156c15a3-3406-45d3-a25e-27179baf0c59"',
+                'output="TissueDissociationProtocol.pdf"',
+                ''
+            ],
+            [
+                f'url="{config.service_endpoint()}/repository/files/5f9b45af-9a26-4b16-a785-7f2d1053dd7c"',
+                'output="SmartSeq2_RTPCR_protocol.pdf"',
+                ''
+            ]
+        ]
+        for expected, actual in zip_longest(expected_body, sorted(sliced(body, 3))):
```

Comment:
Why do you compare each item of the outer list separately?

Reply:
Ah, good point.